lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4354,7 +4354,7 @@ namespace Js
             AssertMsg(false, "Non-matching start of document");
         }
 
-        GetLineCharOffsetFromStartChar(cchStartOffset, &line, &col, false /*canAllocateLineCache*/);
+        GetLineCharOffsetFromStartChar(cchStartOffset, &line, &col, false /*canAllocateLineCache*/, true /*canDumpLibraryCode*/);
```

Comment:
As it is a prototype right now, it's OK to hardcode. Also `canDumpLibraryCode` is an optional argument, by default equals to `false`.